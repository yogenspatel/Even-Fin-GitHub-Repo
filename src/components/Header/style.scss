header {
    background-color: #4388c4;
    height: 48px;
}
header img {
    width: 100px;
}

.heading {
    margin: 50px auto;
}
