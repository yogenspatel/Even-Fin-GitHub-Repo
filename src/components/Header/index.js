/* eslint-disable jsx-a11y/alt-text */
/* eslint-disable jsx-a11y/anchor-is-valid */
import React from 'react';
import './style.scss';

const Header = () => (
  <header className="navbar navbar-expand navbar-dark flex-column flex-md-row bd-navbar">
    <a href="#" className="container">
      <img src="../../../assets/logo.png" />
    </a>
  </header>
);

export default Header;
