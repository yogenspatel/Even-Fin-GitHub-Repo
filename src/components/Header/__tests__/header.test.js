/* eslint-disable no-undef */

import React from "react";
import ReactDOM from "react-dom";
import Header from "../../Header";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";

it("Header renders without crashing", () => {
  const header = document.createElement("header");
  ReactDOM.render(<Header />, header);
});

it("Header Match Snapshot", () => {
  const wrapper = shallow(<Header />);
  expect(toJson(wrapper)).toMatchSnapshot();
});
