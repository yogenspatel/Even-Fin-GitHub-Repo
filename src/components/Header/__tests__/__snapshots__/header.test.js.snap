// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Match Snapshot 1`] = `
<header
  className="navbar navbar-expand navbar-dark flex-column flex-md-row bd-navbar"
>
  <a
    className="container"
    href="#"
  >
    <img
      src="../../../assets/logo.png"
    />
  </a>
</header>
`;
